### PR TITLE
add some docs to rust

### PR DESCRIPTION
**Pull request by svrnm**
No a rust expert, but I used some of the existing examples to get this very basic getting started example together. Hope it makes sense ;-)

cc @open-telemetry/rust-approvers

### CODE REVIEW

File: content/en/docs/instrumentation/rust/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+use opentelemetry::sdk::trace::TracerProvider;
+use opentelemetry::{global, sdk::propagation::TraceContextPropagator, trace::Tracer};
+use opentelemetry_stdout::SpanExporter;
+
+async fn handle(req: Request<Body>) -> Result<Response<Body>, Infallible> {
+    let mut response = Response::new(Body::empty());
+
+    let tracer = global::tracer("dice_server");
+
+    let _span = tracer
```
[the remark this answers is not in the document]
Comment (svrnm) [reply content/en/docs/instrumentation/rust/getting-started.md:132]: sure, let me take a look

[the remark this answers is not in the document]
Comment (svrnm) [reply content/en/docs/instrumentation/rust/getting-started.md]: added

File: content/en/docs/instrumentation/rust/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+$ cargo run --bin dice_server
+...
+Listening on 127.0.0.1:8080
+```
+
+When you send a request to the server at <http://localhost:8080/rolldice>,
+you'll see a span being emitted to the console:
+
+```json
+{"resourceSpans":[{"resource":{"attributes":[{"key":"service.name","value":{"stringValue":"unknown_service"}}]},"scopeSpans":[{"scope":{"name":"dice_server"},"spans":[{"traceId":"d77a477e02a76455b38077d67f955589","spanId":"8bf4ecb5d0a7d429","parentSpanId":"","name":"GET /rolldice2","kind":2,"startTimeUnixNano":1691074020952963000,"endTimeUnixNano":1691074020952968000,"attributes":[],"droppedAttributesCount":0,"droppedEventsCount":0,"droppedLinksCount":0,"status":{"message":"Not Found","code":1}}]}]}]}
```
[the remark this answers is not in the document]
Comment (chalin) [reply content/en/docs/instrumentation/rust/getting-started.md:190]: Ah, right I see. Sure, leaving as is, or pretty printing + note works for me. Your call.

[the remark this answers is not in the document]
Comment (svrnm) [reply content/en/docs/instrumentation/rust/getting-started.md]: > pretty printing + note
done.

**Review by chalin — APPROVED** [review]

File: content/en/docs/instrumentation/rust/exporters.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+[Zipkin](https://zipkin.io/). OpenTelemetry Rust provides exporters for some
+common open source backends.
+
+Below you will find some introductions on how to set up backends and the
+matching exporters.
+
+## OTLP endpoint
+
+To send trace data to a OTLP endpoint (like the [collector](/docs/collector) or
+Jaeger) you'll want to use an exporter crate, such as `opentelemetry_otlp`, e.g.
```
Comment (chalin) [inline content/en/docs/instrumentation/rust/exporters.md:18]:
```suggestion
Jaeger) you'll want to use an exporter crate, such as `opentelemetry_otlp`:
```

File: content/en/docs/instrumentation/rust/exporters.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+```shell
```
Comment (chalin) [inline content/en/docs/instrumentation/rust/exporters.md:20]: Is this `shell` ... or `toml`, or something else?

File: content/en/docs/instrumentation/rust/exporters.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+[dependencies]
+opentelemetry-otlp = { version = "0.13", features = ["default"] }
+```
+
+Next, configure the exporter to point at an OTLP endpoint. For example you can
+update `init_tracer` in `dice_server.rs` from the
+[Getting Started](/docs/instrumentation/rust/getting-started/) like the
```
Comment (chalin) [inline content/en/docs/instrumentation/rust/exporters.md:27]:
```suggestion
[Getting Started](../getting-started/) like the
```

File: content/en/docs/instrumentation/rust/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+  ]
+}
+```
+
+## What next?
+
+For more:
+
+- Read the [documentation for Rust API & SDK](/docs/instrumentation/rust/api)
+- try out more [examples](/docs/instrumentation/rust/examples/).
```
Comment (chalin) [inline content/en/docs/instrumentation/rust/getting-started.md:239]:
```suggestion
- Try other [examples](../examples/).
```

File: content/en/docs/instrumentation/rust/getting-started.md [hunk 7]
```diff
@@ -0,0 +1,241 @@
...
+    }
+  ]
+}
+```
+
+## What next?
+
+For more:
+
+- Read the [documentation for Rust API & SDK](/docs/instrumentation/rust/api)
```
Comment (chalin) [inline content/en/docs/instrumentation/rust/getting-started.md:238]:
```suggestion
- Consult the [API & SDK](../api/) reference
```

File: content/en/docs/instrumentation/rust/getting-started.md [hunk 8]
```diff
@@ -0,0 +1,241 @@
...
+## Example Application
+
+The following example uses a basic [hyper](https://hyper.rs/) application. If
+you are not using hyper, that's OK — you can use OpenTelemetry Rust with other
+HTTP implementations as well, such as Actix Web and Tide. For a complete list of
+libraries for supported frameworks, see the
+[registry](/ecosystem/registry/?component=instrumentation&language=rust).
+
+For more elaborate examples, see
+[examples](/docs/instrumentation/rust/examples/).
```
Comment (chalin) [inline content/en/docs/instrumentation/rust/getting-started.md:28]:
```suggestion
[examples](../examples/).
```

**Review by chalin — APPROVED** [review]
A few more copyedits.

**Review by cijothomas — APPROVED** [review]
LGTM.

**Comment by chalin** [comment]
Thanks @svrnm et al.!

**Comment by hdost** [comment]
Thanks for the contribution!

**Pull request merged**